policy-engine/src/graph.rs

```diff
@@ -56,6 +56,7 @@ pub(crate) fn index(
         HTTP_GRAPH_BAD_REQS.inc();
         return Box::new(future::err(e));
     }
+    let plugin_params = req.query().to_owned();
```

Comment:
Leftover?

Reply:
Good catch. I haven't converted the policy-engine to using the plugins yet. We don't actually have any plugins which have to run in the policy-engine just yet, but I'll prepare the infrastructure for it within the scope of this PR.

Reply:
This one is in use again, even though the consume is a no-op.